zerver/middleware.py

```diff
@@ -210,8 +210,7 @@ def write_log_line(log_data: MutableMapping[str, Any], path: str, method: str, r
         logger.info(logger_line)
 
     if (is_slow_query(time_delta, path)):
-        queue_json_publish("slow_queries", dict(
-            query="%s (%s)" % (logger_line, requestor_for_logs)))
+        slow_query_logger.info(logger_line)
```

Comment:
Is dropping `requestor_for_logs` intentional?

Reply:
Yes, requestor_for_logs is already in `logger_line`, so it was redundant to have it here.